geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/ResultModel.java

```diff
@@ -178,7 +181,51 @@ public ResultModel createCommandProcessingError(String message) {
   public ResultModel createError(String message) {
     addInfo().addLine(message);
     setStatus(Result.Status.ERROR);
+    return this;
+  }
 
+  public ResultModel createInfo(String message) {
+    addInfo().addLine(message);
+    setStatus(Result.Status.OK);
     return this;
   }
+
+  public ResultModel buildResult(List<CliFunctionResult> functionResults) {
+    return buildResult(functionResults, null, null);
+  }
+
+  public ResultModel buildResult(List<CliFunctionResult> functionResults, String header,
```

Comment:
I think this should be more descriptive - something like `buildMemberStatusResults`